com.unity.robotics.urdf-importer/Runtime/AssetHandlers/UrdfAssetPathHandler.cs

```diff
@@ -87,27 +89,19 @@ public static string GetFullAssetPath(string relativePath)
 
         public static string GetRelativeAssetPathFromUrdfPath(string urdfPath, bool convertToPrefab=true)
         {
-            if (!urdfPath.StartsWith(@"package://"))
-            {
-               Debug.LogWarning(@$"{urdfPath} is not a valid URDF package file path. Path should start with package://, and URDF file should be in the directory root.");
-               if (urdfPath.Substring(0, 3) == "../")
-               {
-                   Debug.LogWarning("Attempting to replace file path's starting instance of `../` with standard package notation `package://` to prevent manual path traversal at root of directory!");
-                   urdfPath = $@"package://{urdfPath.Substring(3)}";
-               }
-               else
-               {
-                   return null;
-               }
-            }
             string path;
             if (urdfPath.StartsWith(@"package://"))
             {
                 path = urdfPath.Substring(10).SetSeparatorChar();
             }
+            if (urdfPath.StartsWith(@"file://"))
```

Comment:
could you add a description how you tested your changes? Maybe a screenshot/gif to show that you can import a urdf file with the new `file://` URI? 

I have a comment in the original issue reported. https://github.com/Unity-Technologies/URDF-Importer/issues/146#issuecomment-924489678 

You can also use https://github.com/Unity-Technologies/URDF-Importer/blob/main/com.unity.robotics.urdf-importer/Runtime/RuntimeImport/RuntimeUrdfImporterExample.cs for importing local urdf files.

Reply:
Thank you, I tested on Runtime URDF Importer and succeed to spawn gundam. (https://github.com/gundam-global-challenge/gundam_robot)
![Screenshot from 2021-09-24 19-12-38](https://user-images.githubusercontent.com/10348912/134773682-b8e3d6d6-1eb8-4137-8364-5bff10a0c0b6.png)

Reply:
I add description comment at https://github.com/Unity-Technologies/URDF-Importer/pull/147/commits/b971bf0a0f70dd20f71acc59be16dbaf763f2360

Reply:
URDF file of the gundam robot is here.
URDF was not supported by github, so I renamed file from output.urdf to output.urdf.txt.
[output.urdf.txt](https://github.com/Unity-Technologies/URDF-Importer/files/7229835/output.urdf.txt)

Reply:
This URDF file was generated by our tool https://github.com/OUXT-Polaris/unisim_ros2_control.
Because of C# cannot resolve ROS2 package path and find installed ROS2 resources that is referenced from URDF.

Reply:
If I understand correctly, you only replaced the `package://` with `file://<absolute path to mesh files>` in the urdf file itself. Is there anything else that you changed in https://github.com/gundam-global-challenge/gundam_robot/tree/master/gundam_rx78_description?

Another question, do you have any use case to load urdf with `file://` uri in Edit mode? Thanks.

Reply:
I am now integrating Unity with ROS2 toolchain, and currently, I did not considering this usecase.